csrankings.csv

```diff
@@ -8967,6 +8997,7 @@ Jon Whittle,Monash University,https://research.monash.edu/en/persons/jon-whittle
 Jon Youn,University of Nebraska - Omaha,https://www.cune.edu,NOSCHOLARPAGE
 Jonas Almström Duregård,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/jonas-duregard.aspx,NOSCHOLARPAGE
 Jonas Kuhn,University of Stuttgart,http://www.ims.uni-stuttgart.de/institut/mitarbeiter/jonas,t5vqVEkAAAAJ
+Jonas Teuwen,Radboud University,http://diagnijmegen.nl/index.php/Person?name=Jonas_Teuwen,NOSCHOLARPAGE

```

Comment:
This person does not appear to be in CS.

Reply:
> This person does not appear to be in CS.
Thanks for your checking. Actually, at Radboud University, we have an organization called Radboud AI, which involves researchers from different sections, such as cognitive neural science, machine learning, medical imaging, language technology...(more details in https://www.ru.nl/ai/research-education/research-groups/). Evidently, most researchers I updated have already published papers in top AI conferences such as NeurIPS, ACL, SIGIR, etc.. But you are right that the people you have spotted have no publications in those conferences so far. I suppose they supervise PhD students in the related topics, but not sure at this moment. You can remove them so.

Reply:
Please remove these faculty - thanks.

Reply:
Note that pubs in the conferences aren't the deciding factor - it's whether or not they can solely supervise PhDs in computer science.

Reply:
> Please remove these faculty - thanks.

Hi, I just removed 9 people (from medical areas) who are not eligible with respect to the policies as you pointed out. The rest of people are eligible now.